src/App.test.js

```diff
@@ -653,12 +666,104 @@ describe('App component', () => {
         expect(localStorageItems['dopplerFirstOrigin.value']).toBeDefined();
         expect(localStorageItems['dopplerFirstOrigin.value']).toEqual('testOrigin');
         expect(localStorageItems['dopplerFirstOrigin.date']).toBeDefined();
+        expect(localStorageItems['UtmCookies']).toBeDefined();
         const dopplerOriginDate = new Date(localStorageItems['dopplerFirstOrigin.date']);
         expect(dopplerOriginDate).toBeDefined();
         expect(dopplerOriginDate.getFullYear()).toBeGreaterThan(2018);
       });
     });
 
+    it('check utm parameters', async () => {
```

Comment:
How about being a bit more descriptive? Like this:
```suggestion
    it('should check utm parameters are stored', async () => {
```